src/main/resources/application.yml

```diff
@@ -182,3 +182,6 @@ cloud-storage:
     multiplier: 2
     max: 3500
     max-attempts: 3
+
+# This passphrase is used to encrypt the responseId in EQ
+passPhrase: CENSUS
```

Comment:
On its own it is not clear what this attribute applies to.
I think this should be under the "eq" hierarchy, since it passed into the eq-launcher. something like
```
eq:
  host: www.google.com
  response-id-salt: CENSUS
```
... then it won't require a comments since it is obvious by the name.

Reply:
also please add a line at the end of this file, so that the github red-circle doesn't appear